scripts/ci_checks/linux-x86_64/alpine.sh

```diff
@@ -8,5 +8,5 @@ scons -Q \
       --enable-benchmarks \
       --enable-examples \
       --enable-doxygen \
-      --build-3rdparty=openfec,cpputest,google-benchmark \
+      --build-3rdparty=openfec,cpputest,google-benchmark,sndfile \
       test
```

Comment:
Scripts in `ci_checks/linux-x86_64` and ``ci_checks/linux-checks` use docker images as described here: https://roc-streaming.org/toolkit/docs/development/continuous_integration.html

The idea is to test build on different distros. So instead of using --build-3rdparty for all dependencies, we use pre-installed dependencies from distro as much as possible, and use --build-3rdparty only for dependencies missing in distro.

That's why here you see, for example openfec (it's not present in alpine), but don't see, for example, speexdsp (it's present).

I think libsndfile is available in all or most distros. So, instead of adding it to --build-3rdparty, we should add it to docker images here: https://github.com/roc-streaming/dockerfiles

Here is example PR that added a dependency to all images: https://github.com/roc-streaming/dockerfiles/pull/12/files